Source/Core/Core/HW/DSPHLE/UCodes/AXVoice.h

```diff
@@ -419,9 +447,16 @@ void ProcessVoice(PB_TYPE& pb, const AXBuffers& buffers, u16 count, AXMixControl
   // Optionally, execute a low pass filter
   if (pb.lpf.enabled)
   {
-    pb.lpf.yn1 = LowPassFilter(samples, count, pb.lpf.yn1, pb.lpf.a0, pb.lpf.b0);
+    LowPassFilter(samples, count, pb.lpf);
   }
 
+#ifdef AX_WII
+  if (pb.biquad.on)
```

Comment:
Is `on` set to 2 here, or 1?

Reply:
Don't know, but the filter is run when `on` is non-zero.

Reply:
I think it might be better to use an explicit `!= 0` check then, and maybe adjust the comment on `PBBiquadFilter` if needed.  If it's only set to 2 for the remote version (which would make sense for distinguishing between the low-pass and biquad filters in the same memory) then the comment should be moved to `PBInfImpulseResponseWM`.